### DISCUSSION TITLE
Upload image sequence

### DISCUSSION THREAD

**Discussion by inesqvieira** [talmolab/sleap#1156] [Help!]
Hi, new SLEAP user here. 
Is it possible to load an image sequence as the original frames? 
I can always convert my sequences to video format but I would like to avoid it...
Thank you

**Comment by roomrys** [comment]
Hi @inesqvieira,

We allow users to import a collection of images if they are part of a special format (DLC, Coco, etc) or if the images are stored within an HDF5 dataset or [npy](https://numpy.org/doc/stable/reference/generated/numpy.save.html) file. We should develop a feature to allow users to import a directory of images (in the same sequence they are listed in the directory - alphabetically). This is similar to how we handle importing DLC datasets, so some of the code is already written.

For you case, [converting to a video via ffmpeg](https://stackoverflow.com/questions/24961127/how-to-create-a-video-from-images-with-ffmpeg) might be the easiest option until we integrate a feature for loading a "video" from a directory of images.

Let us know if this helps or if you need any help!

Thanks,
Liezl
